app/src/main/java/com/github/orkest/View/EditProfileActivity.kt

```diff
@@ -0,0 +1,221 @@
+package com.github.orkest.View
+
+import android.net.Uri
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.rememberLauncherForActivityResult
+import androidx.activity.compose.setContent
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberImagePainter
+import com.github.orkest.View.ui.theme.OrkestTheme
+import com.github.orkest.R
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.graphics.Color
+
+class EditProfileActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            editProfileSetting {
+                EditProfileScreen()
+            }
+        }
+    }
+}
+
+/**
+ * Function generating the screen
+ */
+@Composable
+fun editProfileSetting(content: @Composable () -> Unit) {
+    OrkestTheme {
+        // A surface container using the 'background' color from the theme
+        Surface(
+            modifier = Modifier.fillMaxSize(),
+            color = MaterialTheme.colorScheme.background
+        ) {
+            content()
+        }
+    }
+}
+
+/**
+ * Principal function in which we build the general structure of the activity
+ */
+@Composable
+fun EditProfileScreen() {
```

Comment:
Maybe you could put this composable function into its own file to make it more modular and rename the activity into something like ProfileActivity (or wait for @saraanej 's part) so that we only get one activity for the profile that navigates between the different needed pages. But this is a design choice that can be discussed further in the merging meeting.

Reply:
Yes, I'd rather we discuss it during a meeting.